wcomponents-theme/src/test/spec/wc.file.getFileSize.test.mjs

```diff
@@ -13,5 +13,10 @@ describe("wc/file/getFileSize", () => {
 		addFilesToInput(element, [{ value: "SixteenChars.gif", type: "image/gif" }]);
 		expect(getFileSize(element)[0]).toBe(16);
 	});
+	/*it("Throw type error", () => {
```

Comment:
What's this one?
I would suggest making a decision, either delete it or reinstate it.